docs/README.md

```diff
@@ -6,8 +6,6 @@ Liionsden is a Web application comprising a database of electrochemical (Lithium
 Ideas drawn from:
 
 * 'Galvanalyser' by Luke Pitt and Prof. David Howey
-* [liionsden by Ivan Kotegov at Imperial College](https://github.com/FaradayInstitution/liionsden)
 * [Universal Battery Database by Samuel Buteau](https://github.com/Samuel-Buteau/universal-battery-database)
-* [dfndb](https://github.com/ndrewwang/dfndb) by @ndrewwang
```

Comment:
That repo does not seem to exist, but then I thought it might be private. I'll leave it there, indicate that is private and tell the link checker not to check it.